tests/queries/0_stateless/03203_generateSnowflakeID.sql

```diff
@@ -0,0 +1,11 @@
+-- Test generateSnowflakeID without machine_id
```

Comment:
Please extend existing tests tests/queries/0_stateless/03130_generateSnowflakeId.sql instead, thanks!

Reply:
Fixed, please review again, thank you!